example/ios/Flutter/AppFrameworkInfo.plist

```diff
@@ -21,6 +21,6 @@
   <key>CFBundleVersion</key>
   <string>1.0</string>
   <key>MinimumOSVersion</key>
-  <string>11.0</string>
+  <string>12.0</string>
```

Comment:
@Subhash703 - can you mention this in changelog as well? Seems pretty significant to be mentioned

Reply:
@harshgarg18 this is in example app